src/reynir/verbframe.py

```diff
@@ -107,14 +107,16 @@ def add_error(
         corr: str,
     ) -> None:
         """ Take note of a verb object specification with an $error pragma """
-        VerbErrors.check_args(args)
         corrlist = corr.split(",")
         errlist = corrlist[0].split("-")
         errkind = errlist[0].strip()
         verb_with_cases = "_".join([verb] + args)
         if corrlist[0] == "OBJ-CASE":
+            VerbErrors.check_args(args)
             VerbErrors.OBJ_ERRORS[verb][args[0]] = corrlist[1].strip()
-        if errkind == "OBJ":
+        
+        """
```

Comment:
Better to use a vertical column of # marks to comment out code (easy to add and delete in VSCode using multi-line selections, Ctrl+Alt+up/down)

Reply:
Fixed; unfortunately Ubuntu seems to override that keyboard shortcut :(